src/lib/loaders/loaders_with_authentication/vortex.ts

```diff
@@ -34,15 +36,98 @@ export default (accessToken, opts) => {
     { method: "POST" }
   )
 
+  const vortexGraphqlLoader = ({ query, variables }: LoaderArgs) =>
+    vortexLoader(
+      "/graphql",
+      { query, variables: JSON.stringify(variables) },
+      {
+        method: "POST",
+      }
+    )
+
   return {
     vortexTokenLoader,
-    vortexGraphqlLoader: ({ query, variables }: LoaderArgs) =>
-      vortexLoader(
-        "/graphql",
-        { query, variables: JSON.stringify(variables) },
-        {
-          method: "POST",
+    vortexGraphqlLoader,
+    marketPriceInsightsBatchLoader: async (
```

Comment:
I have moved the logic from `loadBatchPriceInsights` here because the functionality is basically just a loader.